modules/proxy-namespace-procedure.adoc

```diff
@@ -0,0 +1,42 @@
+
+[[red-hat-quay-proxy-namespace-procedure]]
+= Using {productname} to proxy a namespace in a registry
+
+The following procedure describes how you can use {productname} to proxy a namespace in a registry. This procedure is set up to proxy quay.io, which allows users to use `podman` to pull any public image from any namespace on quay.io.
```

Comment:
In both this sentence and on the title right above it we're saying that this document will show how to proxy a namespace in a registry.
The examples then show how to proxy an entire registry, and in the notes is where we mention that you can also proxy a single namespace.
This feels a little confusing to me, do you think we can make this more consistent?